package.json

```diff
@@ -52,6 +52,7 @@
   },
   "homepage": "https://github.com/contentful/live-preview#readme",
   "dependencies": {
+    "lodash": "^4.17.21",
```

Comment:
should we maybe only import `lodash.isEqual`? instead of having the whole lodash a dependency